entities.py

```diff
@@ -124,11 +124,11 @@ def use_card(self, card, target: 'Enemy', exhaust, pile) -> None:
             ansiprint("<red>This card is not playable. This message shouldn't appear outside of tests.</red>")
             return
         if card.target == 'Single':
-            card.apply(self, target)
+            card.apply(origin=self, target=target)
         elif card.target == 'Area':
-            card.apply(self, active_enemies)
+            card.apply(origin=self, enemies=active_enemies)
         elif card.target == 'Player':
-            card.apply(self)
+            card.apply(origin=self)
```

Comment:
I suggest using keyword args in general, especially at the beginning when functions are changing often. If you rely just on the order of args, that can be more brittle.

Reply:
Fair. I've come across problems where I change the order of a function's arguments and lots of things break because of it.